znnl/loss_functions/__init__.py

```diff
@@ -41,4 +49,10 @@
     MeanPowerLoss.__name__,
     SimpleLoss.__name__,
     CrossEntropyLoss.__name__,
+    WassersteinLoss.__name__,
```

Comment:
Not sure how I feel about this if you can't use it to train a network. I feel like there will be a lot of avoidable issues

Reply:
Done in 
[da01395](https://github.com/zincware/ZnNL/pull/116/commits/da01395f4712b503ba946fa02c6bfe4276767f17)